src/main/java/com/box/sdk/Metadata.java

```diff
@@ -0,0 +1,163 @@
+package com.box.sdk;
+
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+
+/**
+ * The Metadata class represents one type instance of Box metadata.
+ *
+ * Learn more about Box metadata:
+ * https://developers.box.com/metadata-api/
+ */
+public class Metadata {
+    private final JsonObject values;
+    private JsonArray operations;
+
+    /**
+     * Creates an empty metadata.
+     */
+    public Metadata() {
+        this.values = new JsonObject();
+    }
+
+    /**
+     * Creates a new metadata.
+     * @param values The initial metadata values.
+     */
+    public Metadata(JsonObject values) {
```

Comment:
We try not to expose the minimal-json types in the SDK's API and instead prefer plain JSON strings, such as:

```java
public Metadata(String json) { ... }
```

If other parts of the SDK need this constructor for performance reasons (so that JSON doesn't have to be re-parsed) then it can be made package-private.